src/libs/actions/Report.js

```diff
@@ -1506,11 +1505,6 @@ function createPolicyRoom(policyID, reportName, visibility) {
                 Log.error('Unable to grab policy room after creation', reportID);
                 return;
             }
-
-            // Make sure the report has its icons set
-            const report = allReports[reportID];
-            const icons = OptionsListUtils.getReportIcons(report, {});
-            Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {icons});
```

Comment:
**clarification:** we are not saving the icons because we build them from personal details now? Or something else?